waspc/cli/Wasp/Cli/Common.hs

```diff
@@ -44,3 +45,6 @@ extCodeDirInWaspProjectDir = [reldir|ext|]
 
 waspSays :: String -> IO ()
 waspSays what = putStrLn $ Term.applyStyles [Term.Yellow] what
+
+waspScreams :: String -> IO ()
+waspScreams what = putStrLn $ Term.applyStyles [Term.Red] what
```

Comment:
I figured it would be nice to be able to display errors in CLI in red vs normal yellow, so anytime we to not throw an error from CLI but still want to report an error, we can scream. :)

Reply:
Yeah I was thinking this would be nice for any CLI cases where we handle/show errors, but do not throw `CommandError`s